index.html

```diff
@@ -34,9 +34,9 @@ <h1 class="logo">Relaxr</h1>
 						<img src="images/blog_photo1.jpg" alt="Crew Meet">
 						<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce varius augue nec metus viverra interdum. Sed in gravida lectus. Pellentesque habitant morbi tristique senectus et netu et malesuada fames ac turpis egestas. Mauris ut vulputate nisl.  Maecenas quis auctor dui. Ut porta aliquam lorem, at volutpat orci commodo a. Suspendisse nunc orci, porttitor ac sapien a, rutrum porttitor odio. Phasellus faucibus tristique orci eget dictum. Proin tincidunt vehicula leo sed mattis. Nunc faucibus augue a pellentesque tristique. Proin suscipit in dui ut convallis.</p>
 
-						<p class="read-more"><a>Read more &gt;</a></p>
+						<p class="readmore"><a>Read more &gt;</a></p>
```

Comment:
In general, we need some way to easily see the different words that are in a class name. We can do this with any number of methods, so long as we apply the naming scheme consistently. So, for this one, instead of `readmore` we would want to use something like `read-more` or `read_more` or `readMore`. Dashes are not ok for JavaScript variables or function names, but are perfectly fine for class names.